support-frontend/app/wiring/Services.scala

```diff
@@ -12,14 +13,14 @@ import controllers.DynamoTableAsyncForUser
 import play.api.BuiltInComponentsFromContext
 import play.api.libs.ws.ahc.AhcWSComponents
 import services._
-import services.aws.AwsS3Client.s3
```

Comment:
I'm not sure why we have an implicit s3 client, but we do, so I just went with it for now!